vignettes/download_jira_data.Rmd

```diff
@@ -17,6 +17,12 @@ seed <- 1
 set.seed(seed)
 ```
 
+Some packages might not be downloadable by install.packages(“packages_name”), we can try to install the package directly from the github
```

Comment:
This seems helpful info in general, rather than specific to this notebook regarding installing R package. Could you add this to https://github.com/sailuh/kaiaulu/wiki/Dependencies-Troubleshooting instead?